src/main/groovy/net/neoforged/gradleutils/GradleUtilsExtension.groovy

```diff
@@ -47,6 +55,7 @@ abstract class GradleUtilsExtension {
 
     @Inject
     GradleUtilsExtension(Project project, ProjectLayout layout, ObjectFactory objects, ProviderFactory providers) {
+        this.project = project
```

Comment:
This is gonna need some testing -- last time I was testing things, the configuration cache didn't exactly like storing a `Project` as an instance field in this extension.

Reply:
Making the field `transient` fixes any configuration cache issues as that field will simply don't be serialized. Which is fine as we only use it during configuration.